cognite/neat/_state/README.md

```diff
@@ -0,0 +1,22 @@
+# NeatState
+
+The neat state controls the `NeatGraphStore` and `NeatRuleStore`. It is implementing a state machine pattern
```

Comment:
Can you just mention that's this is under development? Just to be clear for people that might be checking the repo docs :)